File: javascript/8-kyu/holiday-vi-shark-pontoon/solution.js
function shark(pontoonDistance, sharkDistance, youSpeed, sharkSpeed, dolphin) {
  let timeToBoat = pontoonDistance / youSpeed;

  let timeUntilEaten = sharkDistance / (dolphin ? sharkSpeed / 2 : sharkSpeed);

  return timeToBoat < timeUntilEaten ? "Alive!" : "Shark Bait!";
}

File: csharp/8-kyu/holiday-vi-shark-pontoon/solution.cs
public class Kata
{
    public static string Shark(
      int pontoonDistance, 
      int sharkDistance, 
      int yourSpeed, 
      int sharkSpeed, 
      bool dolphin)
    {
        var timeToBoat = pontoonDistance / yourSpeed;
        var timeUntilEaten = sharkDistance / (dolphin ? sharkSpeed / 2 : sharkSpeed); 
        
        return (timeToBoat < timeUntilEaten) ? "Alive!" : "Shark Bait!";
    }
}